src/components/Switch/switch.component.css

```diff
@@ -0,0 +1,51 @@
+.inputWrapper {
+    position: relative;
+    top: 5px;
+    left: 5px;
+}
+
+.input {
+    position: absolute;
+    left: -9999px;
+    top: -99999px;

```

Comment:
Can you think of a less invasive way to hide the `.input` element from view? This is a functional solution, but it's not ideal to position elements like this. It has disadvantages for things like accessibility and sometimes SEO

Reply:
I thought `display:none` will affect the SEO but after a quick googling they said it doesn't. What do you think?

Reply:
`display: none;` is better for SEO than moving something off-screen without any indication that it's hidden.